doc-content/enterprise-only/admin-and-config/business-central-settings-security-policy-con.adoc

```diff
@@ -0,0 +1,24 @@
+[id='business-central-settings-security-policy-con']
+= Security policy
+
+Every application contains a single security policy which is used every time the system checks a permission. The set of permissions assigned to every role and/or group is called the security policy.
+
+The security policy file is stored in the `WEB-INF/classes/security-policy.properties` under the application's `WAR` file structure.
+
+NOTE: If the security policy is not defined then the security management feature is disabled and the application as if all the resources and features are granted by default.
```

Comment:
This note needs to be revised for clarity. I might need more information about what the intent of the note is.